core/src/utils/gesture/index.ts

```diff
@@ -141,7 +142,20 @@ export const createGesture = (config: GestureConfig): Gesture => {
     return true;
   };
 
+  const blurActiveElement = () => {
+    /* tslint:disable-next-line */
+    if (typeof document !== 'undefined') {
```

Comment:
Is there a reason you decided against a more general truthy check?
e.g `if (document) {`

Reply:
`if (typeof document !== 'undefined') {` is a check to see if the script is being run inside a web browser or not. It allows us to support server-side rendering in Ionic Framework.

Reply:
I see! That makes sense, thank you for elaborating!